bigbluebutton-html5/imports/api/audio/client/bridge/sfu-audio-bridge.js

```diff
@@ -509,4 +509,4 @@ export default class SFUAudioBridge extends BaseAudioBridge {
   }
 }
 
-module.exports = SFUAudioBridge;
+// module.exports = SFUAudioBridge;
```

Comment:
Could you please comment (here or inline) why we're disabling the export.

Reply:
we mainly use esm export/import, so this kind of import isn't need anymore and could cause conflicts on compilation